.travis.yml

```diff
@@ -10,8 +10,8 @@ matrix:
       script:
         - rm -f .copyright.hook && wget https://raw.githubusercontent.com/PaddlePaddle/Paddle/develop/.copyright.hook
         - |
-          bash .tools/check_style.sh
-          RESULT=$?; if [ $RESULT -eq 0 ]; then true; else exit 1 ; fi;
+          bash -x .tools/check_style.sh
+          RESULT=$?; if [ $RESULT -eq 0 ]; then echo Passed ; else echo Failed >&2 ; fi;
```

Comment:
Can you take alook at https://github.com/PaddlePaddle/cloud/pull/614? I think it should be better than this?

Reply:
yes, my pr does not fix it well.

Reply:
OK, closing now, thanks!